src/gui/tray/ActivityItem.qml

```diff
@@ -0,0 +1,288 @@
+import QtQml 2.12
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import Style 1.0
+
+MouseArea {
+    id: activityMouseArea
+    enabled: (path !== "" || link !== "")
+    width: Style.trayWindowWidth
+    height: Style.trayWindowHeaderHeight
+    Layout.alignment: Qt.AlignLeft
+    Layout.margins: 2
+    Layout.fillWidth: true
+    Layout.fillHeight: true
+    hoverEnabled: true
+    onClicked: activityModel.triggerDefaultAction(model.index)
+    
+    Rectangle {
+        Layout.fillWidth: true
```

Comment:
you are not inside a layout, please remove the use of Layout attached properties